From b56cfb15c4513e83ffd31e4e7653c572ea0c0395 Mon Sep 17 00:00:00 2001
From: Dominik Picheta <dominik@cloudflare.com>
Date: Tue, 24 Sep 2024 15:30:12 +0000
Subject: Modify where to look for fast_float dependency.

Similar to fp16, this dependency now needs to be downloaded by bazel.

diff --git a/BUILD.bazel b/BUILD.bazel
index bc3c06083bb50475031703c4e7dc498e2d6384f5..cbef1193f598eb1bbb95eee9857ad70efb2f0d3f 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -3840,23 +3840,21 @@ filegroup(
     }),
 )
 
-v8_library(
-  name = "lib_fast_float",
-  srcs = [
-    "third_party/fast_float/src/include/fast_float/ascii_number.h",
-    "third_party/fast_float/src/include/fast_float/bigint.h",
-    "third_party/fast_float/src/include/fast_float/constexpr_feature_detect.h",
-    "third_party/fast_float/src/include/fast_float/decimal_to_binary.h",
-    "third_party/fast_float/src/include/fast_float/digit_comparison.h",
-    "third_party/fast_float/src/include/fast_float/fast_float.h",
-    "third_party/fast_float/src/include/fast_float/fast_table.h",
-    "third_party/fast_float/src/include/fast_float/float_common.h",
-    "third_party/fast_float/src/include/fast_float/parse_number.h",
+cc_library(
+    name = "lib_fast_float",
+    hdrs = [
+        "@fast_float//:include/fast_float/fast_float.h",
+        "@fast_float//:include/fast_float/ascii_number.h",
+        "@fast_float//:include/fast_float/bigint.h",
+        "@fast_float//:include/fast_float/constexpr_feature_detect.h",
+        "@fast_float//:include/fast_float/decimal_to_binary.h",
+        "@fast_float//:include/fast_float/digit_comparison.h",
+        "@fast_float//:include/fast_float/fast_table.h",
+        "@fast_float//:include/fast_float/float_common.h",
+        "@fast_float//:include/fast_float/parse_number.h",
     ],
-  hdrs = [ "third_party/fast_float/src/include/fast_float/fast_float.h" ],
-  includes = [
-    "third_party/fast_float/src/include",
-  ],
+    include_prefix = "third_party/fast_float/src",
+    deps = [],
 )
 
 cc_library(
